config/config.prod.yaml

```diff
@@ -50,6 +52,7 @@
       INCOMPLETE: Closed
       WORKSFORME: Closed
       REOPENED: In Progress
+    resolution_map: *basic-resolution-map
```

Comment:
This references the anchor before it's defined. You need to move the definition here.